app-header/app-header.html

```diff
@@ -314,16 +314,14 @@ <h1 main-title>App name</h1>
     Polymer({
       is: 'app-header',
 
-      behaviors: [
-        Polymer.AppScrollEffectsBehavior,
-        Polymer.AppLayoutBehavior
-      ],
+      behaviors: [Polymer.AppScrollEffectsBehavior, Polymer.AppLayoutBehavior],
 
       properties: {
         /**
          * If true, the header will automatically collapse when scrolling down.
-         * That is, the `sticky` element remains visible when the header is fully condensed
-         * whereas the rest of the elements will collapse below `sticky` element.
+         * That is, the `sticky` element remains visible when the header is fully
+         *condensed whereas the rest of the elements will collapse below `sticky`
+         *element.
```

Comment:
nit: no space between `*` and text. Not a blocker, but curious why linter doesn't handle this.

Reply:
odd.. it should